package.json

```diff
@@ -25,9 +25,9 @@
     "bluebird": "^3.5.3",
     "cesium": "^1.55",
     "fs-extra": "^7.0.1",
-    "gltf-pipeline": "^1.0.6",
+    "gltf-pipeline": "^3.0.5",
```

Comment:
This might have to stay pinned to version 1.x.x for files like `optimizeGlb` to work. Really though the `2.0-tools` branch should be cleaned up and merged.